packaging/nntrainer.spec

```diff
@@ -103,6 +104,13 @@ Requires:	nnstreamer-tensorflow-lite
 Requires:	%{capi_machine_learning_inference}
 %endif # support_nnstreamer_backbone
 
+%if 0%{?support_tflite_interpreter}
+BuildRequires: tensorflow-lite-devel
+BuildRequires: flatbuffers-devel
+
+Requires:	tensorflow-lite
```

Comment:
1. Check package name (and check if you really need a shared lib of tf-lite.)
2. tensorflow2-lite-devel might be better.

Reply:
1. removed.
2. as discussed in https://github.com/nnstreamer/nntrainer/pull/1201#discussion_r643890397